x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/table/group_title_renderers.tsx

```diff
@@ -76,33 +76,25 @@ export const groupTitleRenderers: GroupPanelRenderer<AlertsGroupingAggregation>
   }
 };
 
-export const INTEGRATION_GROUP_RENDERER_LOADING_TEST_ID = 'integration-group-renderer-loading';
 export const INTEGRATION_GROUP_RENDERER_TEST_ID = 'integration-group-renderer';
 export const INTEGRATION_GROUP_RENDERER_INTEGRATION_NAME_TEST_ID =
   'integration-group-renderer-integration-name';
 export const INTEGRATION_GROUP_RENDERER_INTEGRATION_ICON_TEST_ID = 'integration-group-renderer';
-export const SIGNAL_RULE_ID_GROUP_RENDERER_TEST_ID = 'signal-rule-id-group-renderer';
+export const RELATED_INTEGRATION_GROUP_RENDERER_TEST_ID = 'related-integration-group-renderer';
 
 /**
  * Renders an icon and name of an integration.
- * This component needs to be used within the TableSectionContext which provides the installed packages as well as all the rules.
+ * This component needs to be used within the TableSectionContext which provides the installed packages.
  */
 export const IntegrationNameGroupContent = memo<{
   title: string | string[];
 }>(({ title }) => {
-  const { packages, ruleResponse } = useTableSectionContext();
-  const { integration } = useGetIntegrationFromRuleId({
-    packages,
-    ruleId: title,
-    rules: ruleResponse.rules,
-  });
+  const { packages } = useTableSectionContext();
+  const integrationName = Array.isArray(title) ? title[0] : title;
```

Comment:
This should be okay, but we've used this helper in the past https://github.com/elastic/kibana/blob/92acdae34279a8c96b856a742fe440f0f81dae2b/x-pack/solutions/security/plugins/security_solution/common/endpoint/models/ecs_safety_helpers.ts#L28